pkg/tombstone/tombstone.go

```diff
@@ -113,24 +114,24 @@ func Read(graveyard, name string) (*Tombstone, error) {
 	return &t, nil
 }
 
-type EventHandler func(fsnotify.Event)
+type EventHandler func(string, string, fsnotify.Op)
 
 // LoggingEventHandler is an example EventHandler that logs fsnotify events
-func LoggingEventHandler(event fsnotify.Event) {
-	if event.Op&fsnotify.Create == fsnotify.Create {
-		log.Info("Tombstone Watch: file created:", "name", event.Name)
+func LoggingEventHandler(graveyard string, tombstone string, op fsnotify.Op) {
+	if op&fsnotify.Create == fsnotify.Create {
+		log.Printf("Tombstone Watch: file created: %s/%s\n", graveyard, tombstone)
```

Comment:
Why are we not using kubelog here?

Reply:
Oh, right @stephpalis you are going to need to change this patch to incorporate @askore's changes in the previous pr.

Reply:
New PR is up undoing this change and putting that commit in a different PR: https://github.com/acquia/kubexit/pull/3

Will cleanup this PR once that is approved.